scripts/install_sh/install.sh.txt

```diff
@@ -34,7 +41,7 @@ install() {
 
   (@ for val in data.values.products: @)
   echo "Installing (@= val.product @)..."
-  $dl_bin (@= val.github.url@)/releases/download/(@= val.version @)/(@= val.product @)-${binary_type} > /tmp/(@= val.product @)
+  $dl_bin (@= addProtocol(val.github.url) @)/releases/download/(@= val.version @)/(@= val.product @)-${binary_type} > /tmp/(@= val.product @)
```

Comment:
what does val.github.url normally. return?

The logic of `if starts with http do nothing` otherwise add `https` seems odd. I *thought* the problem was that it started with `http` and we wanted it to be `https` ? Maybe I'm misremembering...

Reply:
Based on the current install.sh, I am guessing that `val.github.url` => `github.com/carvel-dev/{product}`. So, we just want to add `https://` to it, but we just want to be sure that it doesn't already have http or https, hence the if condition.

Reply:
We can make the logic more complex and replace `http` for `https` but Praveen noted we are currently storing the URLs without any protocol, this function is just to prevent us from adding a protocol when we have one. But we can definitely make it smarter and force `https`.